pkg/model/components/kubelet.go

```diff
@@ -158,16 +159,8 @@ func (b *KubeletOptionsBuilder) BuildOptions(o interface{}) error {
 	}
 
 	if cloudProvider == fi.CloudProviderVSphere {
-		glog.Info("In kubenet for vsphere cloud provider")
 		clusterSpec.Kubelet.CloudProvider = "vsphere"
 		clusterSpec.Kubelet.HairpinMode = "vmw-bridge"
```

Comment:
This hairpin mode seems is also not working for us rt? shall we remove this one too?